### ISSUE TITLE
Mission Areas - legend colors

### ISSUE THREAD

**Issue by Keristiena** [uno-isqa-8950/uno-cpi#324]
The colors of the mission areas should not change depending on what the map is. Please be consistent - right now the legend color changes.

**Comment by robynlloos** [comment]
Additionally, the color for All Mission Areas is very hard to see with the color used in the background of the filter menu. See example:
![all mission areas - color hard to see-screen shot](https://user-images.githubusercontent.com/44984230/51133149-c91e1c00-17f9-11e9-8ade-4e8b0c73d79b.JPG)

**Comment by EdemD** [comment]
@Ruthie2018  assigned as QA

**Comment by jnabity** [comment]
Community Partner Type Map still in progress. Remaining maps have been updated.

**Comment by vbhat0891** [comment]
Updated for Community Partner Type as well.
![image](https://user-images.githubusercontent.com/26983338/54882380-6cf1fc80-4e27-11e9-9880-1900f3992673.png)

**Comment by Ruthie2018** [comment]
@vbhat0891 Community partner type does not have social justice type

**Comment by vbhat0891** [comment]
@Ruthie2018 Social Justice type is a mission area and not a Community Partner Type. I do see it in the mission area.

**Comment by Ruthie2018** [comment]
yes, you're right. thanks!

**Comment by Ruthie2018** [comment]
https://docs.google.com/spreadsheets/d/196dOL2oxVoO6fEsMGRaHcOOXat2GYU-d5SRRWGd2COg/edit#gid=547627265

@EdemD @vbhat0891 is there anything else I should test for with this ticket?

**Comment by vbhat0891** [comment]
No, only the mission area colors being consistent across 3 maps having community partner data and "All mission area" having a consistent white color .

![image](https://user-images.githubusercontent.com/26983338/55088385-a621ab00-5079-11e9-891a-81aaf1682180.png)
